src/Vlingo.Wire.Tests/Node/MockConfiguration.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System.Collections.Generic;
+using System.Linq;
+using Vlingo.Actors;
+
+namespace Vlingo.Wire.Tests.Node
+{
+    using Vlingo.Wire.Node;
+    
+    public class MockConfiguration : IConfiguration
+    {
+        private readonly ISet<Node> _nodes;
```

Comment:
we haven't used `_` in private variable names so far (except may be variables like mutex- which are not intended for use in implementing logic). Do you want to use `_` notation throughout this project?

Reply:
Yes, I didn't want to change that for actors, common and uuid but I think that using `_ ` is better to spot private fields and doesn't require to use `this` in situations where parameters to a method are named the same way. Also this eliminates another problem which may be overlooked like `variable hiding` .

As you were not against that change when we were discussing, this will be the only change I'm introducing compared to already existing style :)

Reply:
Absolutely fine. I have no problem with any coding style. Just wanted to confirm if this was intentional. I kinda like the `_` prefix for private vars. Good old ReSharper used to suggest that when I was using it.